app/helpers/Intersect.js

```diff
@@ -528,6 +532,13 @@ async function getPageData(authority, page) {
   return pageData || {};
 }
 
+function shouldUpdatePage(page, pageData) {
+  if (pageData.version !== '1.0') {
```

Comment:
```suggestion
  if (pageData.version >= '1.1') {
```

Clarifies what changes occurred in the versioning imo

Reply:
Fair point